docs/modules/rust-guide/pages/multiply-dependency.adoc

```diff
@@ -12,15 +12,15 @@ ifdef::env-github,env-browser[:outfilesuffix:.adoc]
 :sdk-long-name: DFINITY Canister Software Development Kit (SDK)
 :cdk-long-name: DFINITY Canister Development Kit (CDK) for Rust
 
-One common approach to application design is to calculate or store data in one program that you can then use in another program. 
-This ability to share and use functions defined in different application canisters, even if the underlying programs are written in different languages, is an important strategy for building applications to run the {IC}.
+One common approach to Dapp design is to calculate or store data in one smart contract that you can then use in another smart contract. 
+This ability to share and use functions defined in different smart contract canisters, even if the underlying smart contracts are written in different languages, is an important strategy for building Dapps to run the {IC} blockchain network.
 This tutorial provides a basic introduction to how you can write functions in one language—in the example, {native}—then use the data in another—in this case, {lang}.
 
-For this tutorial, both programs are in the same project. 
+For this tutorial, both smart contract canisters are in the same project. 
```

Comment:
At some point you may want to switch to just canister?

Reply:
Took a pass.  PTAL

Reply:
Updated to "canister smart contract" as per discussions.